MantidQt/CustomInterfaces/src/Reflectometry/ReflMainViewPresenter.cpp

```diff
@@ -32,6 +33,14 @@ ReflMainViewPresenter::ReflMainViewPresenter(
     : m_view(mainView), m_tablePresenter(tablePresenter),
       m_progressView(progressView), m_searcher(searcher) {
 
+  // Register this presenter as the m_tablePresenter's outer presenter
+  m_tablePresenter->registerOuterPresenter(this);
```

Comment:
Perfect. With double dispatch, sometimes `accept` is also used for this kind of method name.